Libraries/LibJS/CMakeLists.txt

```diff
@@ -260,5 +260,9 @@ if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "i.86.*")
     target_link_libraries(LibJS PRIVATE atomic)
 endif()
 
+# This flag has no effect on Windows
```

Comment:
In your PR you described it as having an effect though: clang-cl refuses it.

Reply:
It accepts it but through /clang: as it is a GNU style option so needs to be passed to the clang driver.